tools/install.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+from __future__ import print_function
+import os
+import json
+import sys
+import tempfile
+import shutil
+import gzip
+from zipfile import ZipFile
+import re
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+GITHUB_URL = "https://github.com"
+releases_url_html = "https://github.com/denoland/deno/releases/latest"
+
+install_dir = os.path.join(tempfile.gettempdir(), "deno_install")
+
+home = os.path.expanduser("~")
+
+def get_latest_url():
+
+    res = urlopen(releases_url_html)
+
+    html = res.read().decode('utf-8')
+
+    urls = re.findall(r'href=[\'"]?([^\'" >]+)', html)
+
```

Comment:
There's a lot of blank lines - seems unnecessary for reading - can you condense it a bit?

Reply:
Done! I've still have a couple of blank lines but less :D